projects/control-service/projects/base/src/main/java/com/vmware/taurus/security/SecurityConfiguration.java

```diff
@@ -94,6 +99,8 @@ public SecurityConfiguration(
       FeatureFlags featureFlags,
       @Value("${spring.security.oauth2.resourceserver.jwt.jwk-set-uri:}") String jwksUri,
       @Value("${spring.security.oauth2.resourceserver.jwt.issuer-uri:}") String issuer,
+      @Value("${datajobs.authorization.oauth2.resourceserver.jwt.extra-issuer.uris:}")
+          String[] extraIssuers,
```

Comment:
Should we deprecate `spring.security.oauth2.resourceserver.jwt.issuer-uri` and replace it with one property `spring.security.oauth2.resourceserver.jwt.issuer.uris` which is a list ?

Until removed (in future change) we can concatenate both ?

Reply:
That's my main concern with this change and am happy to do it either way, it's just that this is a standard Spring property, and the second approach is also backwards compatible - users wont't need to change helm charts in already deployed instances with just a single issuer - which should be most common use case.

Reply:
My suggestion is to name the new one not 'extra-issue-uris' but simple issue-uris. 

Otherwise for now to keep the previous one (simply write a comment that it is deprecated). 

At some point in the future when we are sure there is no usage of the deprecated one or we make a explicit major release  we can remove it